relay-server/src/actors/processor.rs

```diff
@@ -3340,11 +3381,9 @@ mod tests {
         )
         .unwrap();
 
-        let config = ProjectConfig {
-            datascrubbing_settings,
-            pii_config: Some(pii_config),
-            ..Default::default()
-        };
+        let mut config = ProjectConfig::default();
+        config.datascrubbing_settings = datascrubbing_settings;
+        config.pii_config = Some(pii_config);
```

Comment:
since there are now private fields in `ProjectConfig`, had to change it like this